x-pack/plugins/cross_cluster_replication/public/app/sections/home/home.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { PureComponent, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentHeaderSection,
+  EuiTitle,
+  EuiSpacer,
+  EuiPageContentHeader,
+  EuiButton,
+} from '@elastic/eui';
+
+import routing from '../../services/routing';
+import { AutoFollowPatternList } from './auto_follow_pattern_list';
+import { BASE_PATH } from '../../../../common/constants';
+import { SectionUnauthorized } from '../../components';
+
+export class CrossClusterReplicationHomeUI extends PureComponent {
+  static propTypes = {
+    autoFollowPatterns: PropTypes.array,
+  }
+
+  state = {
+    sectionActive: 'auto-follow'
+  }
+
+  /**
+   * Get the button in top right corner. In phase 1 we only have the auto-follow patterns
+   * but in phase 2 it can be either "Add Follower" or "Add auto-follow pattern"
+   */
+  getButtonHeader() {
```

Comment:
This is also kind of confusing in its current state.  I know this has an eye to the future, but can we make it clearer how it currently functions?

Reply:
👍